StrataToCBMC.lean

```diff
@@ -18,8 +19,10 @@ open Strata
 def main (args : List String) : IO Unit := do
   match args with
   | [file] => do
-    let text ← IO.FS.readFile file
-    let inputCtx := Lean.Parser.mkInputContext text file
+    let text ← Strata.Util.readInputSource file
+    -- Use "<stdin>" as display name for error messages when reading from stdin
+    let displayName := if file == "-" then "<stdin>" else file
```

Comment:
It'd be sort of nice to remove this duplication between the three top-level executables, but it's not a large enough duplication to be really critical.